src/app/views/sidebar/sample-queries/SampleQueries.tsx

```diff
@@ -166,7 +166,8 @@ const SeeMoreQueriesMessageBar = () => {
               componentNames.MICROSOFT_GRAPH_API_REFERENCE_DOCS_LINK
             )
           }
-          href='https://learn.microsoft.com/graph/api/overview?view=graph-rest-1.0'
+          // eslint-disable-next-line max-len
+          href='https://learn.microsoft.com/en-us/graph/graph-explorer/graph-explorer-overview?view=graph-rest-1.0%2F/WT.mc_id=msgraph_inproduct_graphex'
```

Comment:
Here you could use backticks and + to cut the length of the line rather than disabling rule.